tests/ui/run-functional-tests.sh

```diff
@@ -20,4 +20,5 @@ docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.fu
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart worker
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec web make -f Makefile-docker update_deps
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart web
-docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec --user root selenium-firefox tox -e ui-tests
+sudo chown -R circleci:circleci .
```

Comment:
The old command was running as root (`docker-compose ... exec --user root`) which would have already had access, right? So how would this fix it? Maybe I'm misunderstanding what `--user root` does.

Also, for a change like this it helps to put a comment above saying that it's needed for uploading XPI files (or something like that).

Reply:
Huh, ok